GeneratorInterface/Pythia8Interface/plugins/ReweightUserHooks.h

```diff
@@ -37,10 +37,15 @@ class PtHatEmpReweightUserHook : public Pythia8::UserHooks {
            -5.1575514014931e-01,
            5.5951279807561e-02,
            3.5e+02};
-    sigma = [this](double x) -> double {
-      return (p[0] * pow(x, p[2] + p[3] * log(0.01 * x) + p[4] * pow(log(0.01 * x), 2)) *
-              pow(1 - 2 * x / (13000. + p[5]), p[1])) *
-             x;
+    if (tuneName == "CP5Run3")
+      sigma = [this](double x) -> double {
+        return (p[0] * pow(x, p[2] + p[3] * log(0.01 * x) + p[4] * pow(log(0.01 * x), 2)) *
+              pow(1 - 2 * x / (13600. + p[5]), p[1])) * x;
+    else
+      sigma = [this](double x) -> double {
+        return (p[0] * pow(x, p[2] + p[3] * log(0.01 * x) + p[4] * pow(log(0.01 * x), 2)) *
+              pow(1 - 2 * x / (13000. + p[5]), p[1])) * x;
+      
```

Comment:
Maybe simpler
```suggestion
    const double ecms = ( tuneName == "CP5Run3" ? 13600. : 13000.):
    sigma = [this](double x) -> double {
      return (p[0] * pow(x, p[2] + p[3] * log(0.01 * x) + p[4] * pow(log(0.01 * x), 2)) *
                  pow(1 - 2 * x / (ecms + p[5]), p[1])) * x;
```
(+ code format)